src/query/frontend/ast/cypher_main_visitor.cpp

```diff
@@ -392,8 +392,10 @@ antlrcpp::Any CypherMainVisitor::visitLoadCsv(MemgraphCypher::LoadCsvContext *ct
 
   auto *load_csv = storage_->Create<LoadCsv>();
   // handle file name
-  if (ctx->csvFile()->literal()->StringLiteral()) {
+  if (ctx->csvFile()->literal()) {
```

Comment:
is it ok to remove `StringLiteral` check?

Reply:
Yes, if we don't remove if we will get an error if we use parameter instead of `StringLiteral`because `literal` won't exist so we can't call `StringLiteral` from something that doesn't exist.

Reply:
But I'll add another `if  `that checks if given `literal` is `StringLiteral` because now if we run the query for example with `integet` literal we won't get descriptive error message as before.